learn/experimental/vector_search.mdx

```diff
@@ -102,14 +103,86 @@ curl \
 `documentTemplate` is an optional field you can use to customize the data you send to the embedder. It is highly recommended you configure a custom template for your documents.
 
 `model` is mandatory and must indicate a BERT embedding model.
+</Tabs.Content>
+
+<Tabs.Content label="Ollama">
+Use the `embedders` index setting of the [update `/settings` endpoint](/reference/api/settings) to configure one or more embedders for an index:
+
+```sh
+curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "embedders": {
+      "default": {
+        "source": "ollama",
+        "url": "http://localhost:11434/api/embeddings",
+        "apiKey": "OLLAMA_API_KEY",
+        "model": "MODEL_NAME",
+        "documentTemplate": "A document titled '{{doc.title}}' whose description starts with {{doc.overview|truncatewords: 20}}"
```

Comment:
```suggestion
        "documentTemplate": "A document titled {{doc.title}} whose description starts with {{doc.overview|truncatewords: 20}}"
```

BTW, it is not the only code sample part I notice with single quotes that "collide" with the shell single quotes. I advise you to remove them in those examples (it's a nightmare to put single quotes in a single quoted string in a shell expression) 😢